lib/gtfs/block.ex

```diff
@@ -78,6 +78,23 @@ defmodule Gtfs.Block do
     end
   end
 
+  @spec trip_at_time(t(), Util.Time.time_of_day()) :: Trip.t() | nil
```

Comment:
Enum.find will return nil if nothing is found.

Reply:
We can guarantee that something will be found because if there was no trip in progress, we'd have caught it in one of the first two paths of the `cond`.

Reply:
Dialyzer doesn’t know that so the condition would have to be somewhere.